.travis.yml

```diff
@@ -2,6 +2,7 @@ sudo: false
 language: node_js
 node_js:
   - "8.11"
+  - "11.9"
```

Comment:
i checked available nove engines on azure and it looks like 11.9 is not there, 10.14 seems to be the newest available

Reply:
I will change engine tag.